next/docs/iota-sdk/docs/how-tos/nfts/burn-nft.mdx

```diff
@@ -14,7 +14,15 @@ keywords:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This How-To will show you how to burn an NFT.
+When you burn an [NFT](https://wiki.iota.org/shimmer/develop/explanations/what-is-stardust/tokenization/#non-fungible-tokens-nfts),
+you permanently remove it from circulation on the Tangle, rendering it unusable and untransferable.
```

Comment:
It's removed from the ledger state, not from the Tangle (there it's basically removed when pruned)
Also might be interesting to add, that one get the storage deposit back?